docs/JLBP-6.md

```diff
@@ -156,18 +156,18 @@ is to the decision.
 Examples in open source
 -----------------------
 
-**Case 1**
+**Case 1 (Keep same Java package and Maven ID)**
```

Comment:
can probably diowithout parentheses here and below

Reply:
Removed the parentheses.